dev-util/vulkan-headers/vulkan-headers-1.3.236.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=Vulkan-Headers
+inherit cmake-multilib
```

Comment:
We shouldn't need to install these arch-independant headers twice.  Is this an attempt to work around https://github.com/KhronosGroup/Vulkan-Headers/issues/330?

Reply:
Actually, vulkan-headers 1.3.236 doesn't need it, perhaps it could be needed for 9999 but I'm not sure about that.